nls/platform.properties

```diff
@@ -62,7 +62,9 @@ argo.cluster.decision.requeue.title = Requeue time
 argo.cluster.decision.requeue.title.tooltip = Cluster decision resource requeue time in seconds
 argo.server.name = Argo server
 argo.server.placeholder = Select the correct Argo server
-argo.cluster.tooltip = Argo server to deploy Argo app set
+argo.server.exception = Failed to load the Argo server. Make sure the Argo server is installed and the GitOpsCluster resouce is created successfully.
+argo.server.selection.exception = There is no clusterSet bound to the selected Argo namespace. Click the Add cluster sets tab to bind one or more clustersets.
+argo.cluster.tooltip = Argo server to deploy Argo app set. Click the Add cluster sets tab to create new cluster set.
```

Comment:
Change this to:
`Argo server to deploy Argo app set. Click the Add cluster sets tab to create a new cluster set.`